spec/factories/contributors.rb

```diff
@@ -18,6 +16,8 @@
       end
     end
 
-    organization
+    trait :with_new_organization do
+      organization
+    end
```

Comment:
First of all, I wanted to say that I am a huge fan of this improvement!
We definitely benefit from not needlessly creating an organization for **every** contributor!! :heart_eyes: 

I think my idea when I added the organization to the contributors factory was that it should by default assign a contributor to _an_ organization because that is the most used case, where we have a single organization and almost all contributors belong to the organization. It is optional because we want to be able to have contributors, like myself, who would not count in the total number of contributors allowed for an organizations business plan. I must have overlooked that it would create a new organization to assign the contributor to.

I would be in favor of removing this trait, as it is currently not being used anywhere.
I know there are places where we specify in the tests that we want a contributor assigned to a specific organization. If we find the need to create an organization in this way, we can always add back the trait when we actually use it.

What do you think? I'll approve the PR and leave you to remove it unless you want to explain why you prefer to leave it.